css-two-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+h1,
+h3,
+button {
+  font-family: 'Karla', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+}
+
+h1 {
+  text-align: center;
+  letter-spacing: 3px;
+  margin-top: 55px;
+  margin-bottom: 15px;
+  font-size: 40px;
+}
+
+.head {
+  text-align: center;
+  padding-left: 225px;
+  padding-right: 225px;
```

Comment:
Instead of doing this, I would set a `width` on this element, and use `margin: auto` to center it